uptime-kuma/values.yaml

```
@@ -77,9 +82,17 @@ persistence:
   claimNameOverwrite: ""
   sizeLimit: 4Gi
   storageClass: ""
+  annotations: {}
+  labels: {}
 
 nodeSelector: {}
 
 tolerations: []
 
 affinity: {}
+
+testPod:
+  image: docker.io/busybox
+  annotations:
+    kapp.k14s.io/change-rule: upsert after upserting uptime-kuma.app/start
+    kapp.k14s.io/update-strategy: always-replace
```

```suggestion
  image: docker.io/busybox
  annotations: {}
```
personally I would prefere consistency and have a dedicated `registry` property, yet this is just the `testPod` so I am not picky